src/components/welcome-page/Welcome.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
We should create new foldder in
src/pages 
It will store our routes. Welcom page will be one of them.

Reply:
Another thing.  We should use consistent file names.
So inside WelcomePage folder should be WelcomePage.tsx - inconsistent filenames in bigger projects make mess and finding correct files makes harder